### PR TITLE
Backbone-module_CU-j3b2gk

### PR DESCRIPTION
**Pull request by a-wilkes**

### CODE REVIEW

**Comment by a-wilkes** [comment]
Task linked: [CU-j3b2gk Backbone module](https://app.clickup.com/t/j3b2gk)

**Comment by lpboucher** [comment]
Task linked: [CU-j3b2gk Backbone module](https://app.clickup.com/t/j3b2gk)

File: src/app/modules/backbone/backbone.common.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import { AppComponent } from "@app/app.component";
+import { BackbonePageComponent } from "@modules/backbone/pages/backbone/backbone.page";
+import { BackboneComponent } from "@modules/backbone/components/backbone/backbone.component";
+
+export const pageDeclarations: any[] = [
+    BackbonePageComponent,
+];
+
+export const componentDeclarations: any[] = [
+    BackboneComponent
```
Comment (lpboucher) [inline src/app/modules/backbone/backbone.common.ts]: No need for a backbone component here, we will add other components such as header/sidebar, footer

File: src/app/modules/backbone/backbone.common.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+export const providerDeclarations: any[] = [
+];
+
+export const routes: Routes = [
+    {
+        path: "",
+        component: BackbonePageComponent,
+        children: [
+            { path: "", component: AppComponent },
+            { path: "backbone", component: BackboneComponent },
```
Comment (lpboucher) [inline src/app/modules/backbone/backbone.common.ts]: both of these routes are not needed: the appcomponent will not contain any html, but will rather load only a router-outlet. It is bootstrapped automatically on app load so there is no need to add it here.

The children route here will contain routes to components that will apread "around" the sidebar and the footer. They will components like product-list, product-detail etc.

**Review by lpboucher — CHANGES_REQUESTED** [review]
you can remove the whole backbone component and leave children route array empty for now.

**Review by lpboucher — APPROVED** [review]
All good! merging

**Comment by lpboucher** [comment]
merged

**Pull request merged**